webpages/error/reset-settings.js

```diff
@@ -0,0 +1,36 @@
+document.querySelector("#reset-settings-btn").onclick = () => {
+  if (!window.exportedSettings) {
+    const res = confirm(
+      "Are you sure you want to CLEAR ALL SETTINGS? If you haven't exported your settings first, click Cancel."
+    );
+    if (!res) return;
+    const res2 = prompt("Type 123 to confirm. You will lose all your Scratch Addons settings.");
```

Comment:
A better confirmation command might be "YES", which is what some command-line tools do.

Reply:
Or something that can't be ignored like "Delete my data," "Delete my settings," etc.

Reply:
I think I decided to use 123 because if we ever send this for translation, it would avoid issues such as translators accidentally translating "YES" into their language.

Reply:
I think if it is a word like "YES" it should match the localized string too.

Reply:
As a Spanish speaker I can see that go wrong as "si" and "sí" are different strings.

Reply:
Maybe in that case "123" would be the magic command, otherwise it would be a word?

Reply:
Hmm, that sounds confusing.

Reply:
I don't see why "yes" would be better than "123". The only reason the user has to type anything is that they don't click OK without reading the message - the exact sequence of characters doesn't matter.